packages/plugin/fuses/src/util/getElectronExecutablePath.ts

```diff
@@ -9,8 +9,5 @@ type GetElectronExecutablePathParams = {
 };
 
 export function getElectronExecutablePath({ appName, basePath, platform }: GetElectronExecutablePathParams): string {
-  return path.join(
-    basePath,
-    ['darwin', 'mas'].includes(platform) ? path.join('MacOS', appName) : [appName, process.platform === 'win32' ? '.exe' : ''].join('')
-  );
+  return path.join(basePath, ['darwin', 'mas'].includes(platform) ? path.join('MacOS', appName) : [appName, platform === 'win32' ? '.exe' : ''].join(''));
```

Comment:
To be honest, it took me two comparisons to realize what was different in this revision 🤣. Having multiple ternary operators in a single line of code increases the cognitive load, it would be much better if it could be changed to an if statement.

```suggestion
  if (['darwin', 'mas'].includes(platform)) {
    return path.join(basePath, 'MacOS', appName);
  }

  const suffix = platform === 'win32' ? '.exe' : '';
  return path.join(basePath, `${appName}${suffix}`);
```